src/flexmeasures_client/client.py

```diff
@@ -0,0 +1,280 @@
+from __future__ import annotations
+
+import asyncio
+import socket
+from dataclasses import dataclass
+from typing import Any, cast
+
+import async_timeout
+import pandas as pd
+from aiohttp.client import ClientError, ClientSession
+from yarl import URL
+
+from flexmeasures_client.constants import API_VERSIOM, CONTENT_TYPE_HEADERS
+from flexmeasures_client.response_handling import (
+    check_content_type,
+    check_for_status,
+    check_response,
+)
+
+MAX_POLLING_STEPS: int = 10  # seconds
```

Comment:
Not seconds.

Reply:
I thought I already removed this. Anyway, it's gone by now.